src/components/Card/Card.tsx

```diff
@@ -78,7 +80,7 @@ export const Card = React.forwardRef<HTMLDivElement, CardProps>(function Card(pr
         <div
             style={style}
             ref={ref}
-            role={isClickable ? 'button' : undefined}
+            role={isClickable ? 'button' : role}
```

Comment:
It seems a little bit unfair if user sets property `role` and it does not work in some cases

Reply:
What is the purpose of this overall?